{"text": "2024-06-06 06:27:06 | INFO    | __main__:stash:131 | run - {'image': '20231121', 'version': '0.10.1.post2', 'role': 'EpicPlayer', 'headless': True}\n", "record": {"elapsed": {"repr": "0:00:01.589284", "seconds": 1.589284}, "exception": null, "extra": {"image": "20231121", "version": "0.10.1.post2", "role": "EpicPlayer", "headless": true}, "file": {"name": "claim.py", "path": "/home/runner/work/epic-free/epic-free/epic/src/claim.py"}, "function": "stash", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 131, "message": "run", "module": "claim", "name": "__main__", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:27:06.558699+08:00", "timestamp": 1717626426.558699}}}
{"text": "2024-06-06 06:27:16 | INFO    | __main__:claim_epic_games:97 | Try to flush cookie - {'task': 'claim_epic_games'}\n", "record": {"elapsed": {"repr": "0:00:11.200076", "seconds": 11.200076}, "exception": null, "extra": {"task": "claim_epic_games"}, "file": {"name": "claim.py", "path": "/home/runner/work/epic-free/epic-free/epic/src/claim.py"}, "function": "claim_epic_games", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 97, "message": "Try to flush cookie", "module": "claim", "name": "__main__", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:27:16.169491+08:00", "timestamp": 1717626436.169491}}}
{"text": "2024-06-06 06:27:19 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:00:14.342499", "seconds": 14.342499}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-free/epic-free/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:27:19.311914+08:00", "timestamp": 1717626439.311914}}}
{"text": "2024-06-06 06:27:21 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0xc6 in position 3: invalid continuation byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic-free/epic-free/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 613, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0xc6 in position 3: invalid continuation byte\n", "record": {"elapsed": {"repr": "0:00:16.045141", "seconds": 16.045141}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0xc6 in position 3: invalid continuation byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0xc6 in position 3: invalid continuation byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:27:21.014556+08:00", "timestamp": 1717626441.014556}}}
{"text": "2024-06-06 06:27:23 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:00:18.201143", "seconds": 18.201143}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-free/epic-free/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:27:23.170558+08:00", "timestamp": 1717626443.170558}}}
{"text": "2024-06-06 06:27:52 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:00:47.752222", "seconds": 47.752222}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:27:52.721637+08:00", "timestamp": 1717626472.721637}}}
{"text": "2024-06-06 06:27:52 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:00:47.752779", "seconds": 47.752779}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-free/epic-free/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:27:52.722194+08:00", "timestamp": 1717626472.722194}}}
{"text": "2024-06-06 06:28:23 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 0}\n", "record": {"elapsed": {"repr": "0:01:18.845850", "seconds": 78.84585}, "exception": null, "extra": {"task": "authorize", "retry": 0}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-free/epic-free/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:28:23.815265+08:00", "timestamp": 1717626503.815265}}}
{"text": "2024-06-06 06:28:26 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:01:21.686695", "seconds": 81.686695}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-free/epic-free/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:28:26.656110+08:00", "timestamp": 1717626506.65611}}}
{"text": "2024-06-06 06:28:28 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0xb9 in position 0: invalid start byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic-free/epic-free/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 613, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0xb9 in position 0: invalid start byte\n", "record": {"elapsed": {"repr": "0:01:23.401383", "seconds": 83.401383}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0xb9 in position 0: invalid start byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0xb9 in position 0: invalid start byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:28:28.370798+08:00", "timestamp": 1717626508.370798}}}
{"text": "2024-06-06 06:28:30 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:01:25.602592", "seconds": 85.602592}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-free/epic-free/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:28:30.572007+08:00", "timestamp": 1717626510.572007}}}
{"text": "2024-06-06 06:29:00 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:01:55.153616", "seconds": 115.153616}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:29:00.123031+08:00", "timestamp": 1717626540.123031}}}
{"text": "2024-06-06 06:29:00 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:01:55.154287", "seconds": 115.154287}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-free/epic-free/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:29:00.123702+08:00", "timestamp": 1717626540.123702}}}
{"text": "2024-06-06 06:29:31 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 1}\n", "record": {"elapsed": {"repr": "0:02:26.242388", "seconds": 146.242388}, "exception": null, "extra": {"task": "authorize", "retry": 1}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-free/epic-free/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:29:31.211803+08:00", "timestamp": 1717626571.211803}}}
{"text": "2024-06-06 06:29:34 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:02:29.475539", "seconds": 149.475539}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-free/epic-free/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:29:34.444954+08:00", "timestamp": 1717626574.444954}}}
{"text": "2024-06-06 06:29:36 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0x9e in position 9: invalid start byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic-free/epic-free/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 613, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0x9e in position 9: invalid start byte\n", "record": {"elapsed": {"repr": "0:02:31.185507", "seconds": 151.185507}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0x9e in position 9: invalid start byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0x9e in position 9: invalid start byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:29:36.154922+08:00", "timestamp": 1717626576.154922}}}
{"text": "2024-06-06 06:29:38 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:02:33.412113", "seconds": 153.412113}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-free/epic-free/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:29:38.381528+08:00", "timestamp": 1717626578.381528}}}
{"text": "2024-06-06 06:30:07 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:03:02.962544", "seconds": 182.962544}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:30:07.931959+08:00", "timestamp": 1717626607.931959}}}
{"text": "2024-06-06 06:30:07 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:03:02.963103", "seconds": 182.963103}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-free/epic-free/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:30:07.932518+08:00", "timestamp": 1717626607.932518}}}
{"text": "2024-06-06 06:30:39 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 2}\n", "record": {"elapsed": {"repr": "0:03:34.042065", "seconds": 214.042065}, "exception": null, "extra": {"task": "authorize", "retry": 2}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-free/epic-free/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:30:39.011480+08:00", "timestamp": 1717626639.01148}}}
{"text": "2024-06-06 06:30:39 | ERROR    | __main__:run:158 | An error has been caught in function 'run', process 'MainProcess' (3834), thread 'MainThread' (140091655691136): - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic-free/epic-free/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/home/runner/work/epic-free/epic-free/epic/src/claim.py\", line 158, in run\n    await agent.stash()\n  File \"/home/runner/work/epic-free/epic-free/epic/src/claim.py\", line 151, in stash\n    await self.claim_epic_games(context)\n  File \"/home/runner/work/epic-free/epic-free/epic/src/claim.py\", line 98, in claim_epic_games\n    if await epic.authorize(page):\n  File \"/home/runner/work/epic-free/epic-free/epic/src/epic_games/agent.py\", line 269, in authorize\n    raise RuntimeError(f\"Failed to flush token - agent={self.__class__.__name__}\")\nRuntimeError: Failed to flush token - agent=EpicGames\n", "record": {"elapsed": {"repr": "0:03:34.179333", "seconds": 214.179333}, "exception": {"type": "RuntimeError", "value": "Failed to flush token - agent=EpicGames", "traceback": true}, "extra": {}, "file": {"name": "claim.py", "path": "/home/runner/work/epic-free/epic-free/epic/src/claim.py"}, "function": "run", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 158, "message": "An error has been caught in function 'run', process 'MainProcess' (3834), thread 'MainThread' (140091655691136):", "module": "claim", "name": "__main__", "process": {"id": 3834, "name": "MainProcess"}, "thread": {"id": 140091655691136, "name": "MainThread"}, "time": {"repr": "2024-06-06 06:30:39.148748+08:00", "timestamp": 1717626639.148748}}}
